cli/cmd/testdata/grafana_overwrite.yaml

```diff
@@ -1,5 +1,5 @@
 grafana:
   enabled: true
-  name: linkerd-grafana-overwrite
   image:
     name: linkerd-image-overwrite
+    tag: image-tag-overwrite
```

Comment:
Where does `image-tag-overwrite` come from?

Reply:
This comes from the fact that `image.tag` being overwritten, will be saved in `linkerd-config-addons` and thus being useful for upgrades.

Reply:
ah.. so this is an input file.